templates/puppetdb_plugin.yaml.erb

```diff
@@ -0,0 +1,4 @@
+---
+:puppetdb:
```

Comment:
Check the format of this file.. there's the old hash style used by the plugin which is discouraged, and there's the new style which is DB settings.  The latter would be outside of the 'puppetdb' hash, just top-level puppetdb_enabled/address.